src/sentry/options/defaults.py

```diff
@@ -538,3 +538,7 @@
 # region to control silo multi region mode controls
 # It is safe to switch this off or on freely, it is purely for testing features ahead of hybrid cloud rollout.
 register("hc.region-to-control.monolith-publish", default=False)
+
+# Killswitch for sending tagged internal errors to the hybrid cloud test bed dsn
+# Note: A value that is neither 0 nor 1 is regarded as 0
+register("store.use-hc-test-bed-dsn-sample-rate", default=0)
```

Comment:
I feel like it's not really being used as a 'rate' here and more of a bool to act as a killswitch 🤔, also we usually create options first as a separate PR, then have it initialized so it's cached on deploy for when it's accessed.

Reply:
New options will get cached on first retrieval (see https://github.com/getsentry/sentry/pull/39799).  Not sure if that changes us wanting to split out the new option.